### PR TITLE
WEBUI-784: update trigger-workflow action with result output

### PR DESCRIPTION
**Pull request by richardsd**
This PR adds an output message to the `trigger-workflow` action.

### CODE REVIEW

File: .github/workflows/trigger-workflow.yaml [hunk 1]
```diff
@@ -20,14 +20,27 @@ jobs:
...
 
+      # This step will add an output named 'result'
       - name: Trigger workflow with dispatch event
+        id: main
         uses: ./trigger-workflow
         with:
           owner: nuxeo
           repo: ui-team-gh-actions
-          branch-name: main
+          branch-name: WEBUI-784-webui-elements-preview
```
Comment (semisse) [inline .github/workflows/trigger-workflow.yaml]: don't forget to update the branch to `main` 🙏

**Review by semisse — DISMISSED** [review]

**Review by Gabez0r — COMMENTED** [review]
Looks good. I think we could maybe change the name of [.github/workflows/trigger-workflow.yaml](https://github.com/nuxeo/ui-team-gh-actions/pull/5/files#diff-6f615b00c1481cc1c2e3437cd7139989ea4ffc5e0ecada7219384a5bbc584eb3) and maybe
 [.github/workflows/dispatch.yaml](https://github.com/nuxeo/ui-team-gh-actions/pull/5/files#diff-3fe58b4d2f790268b721ba289fe09ab918d12d583b18a4d6cf6e53d452d595bf) for it to be less confusing. Perhaps it could incude the word "test" so that we understand more easily that this is just used for testing purposes.

**Comment by richardsd** [comment]
> Looks good. I think we could maybe change the name of [.github/workflows/trigger-workflow.yaml](https://github.com/nuxeo/ui-team-gh-actions/pull/5/files#diff-6f615b00c1481cc1c2e3437cd7139989ea4ffc5e0ecada7219384a5bbc584eb3) and maybe [.github/workflows/dispatch.yaml](https://github.com/nuxeo/ui-team-gh-actions/pull/5/files#diff-3fe58b4d2f790268b721ba289fe09ab918d12d583b18a4d6cf6e53d452d595bf) for it to be less confusing. Perhaps it could incude the word "test" so that we understand more easily that this is just used for testing purposes.

I agree Gabe, it would be easier to understand what these workflows are used for 👍 Thanks for the feedback.

**Comment by richardsd** [comment]
Please ignore the failing test. This is happening because we renamed the `dispatch` workflow to `dispatch-test`, and GH Actions is looking for that name in the `main` branch.

**Review by Gabez0r — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Pull request merged**